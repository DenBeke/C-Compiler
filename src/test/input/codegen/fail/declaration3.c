// Undeclared...

int test() {
    return 3;
}

int main() {

    int a = 5;
    int b = 3;
    
    int c = a + b;

    a = test() + c + a + b + d;

}