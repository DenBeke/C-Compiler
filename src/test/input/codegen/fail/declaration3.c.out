[31m[ERROR] line 14: Use of undeclared 'd'[0m
